README.org

```diff
@@ -20,6 +36,19 @@
       to attempt to move these spec changes into the official matrix-org spec.
   - Provide a central location for other important documentation on the Matrix
     ecosystem.
+
+* Repositories
+
+The following repositories are currently available with the intended purposes:
+
+  - [[https://github.com/eventually-matrix/matrix-doc][matrix-doc]]: Document current
```

Comment:
I had conceptualized our fork of matrix-doc as being more of a staging area
for upstreaming, but I can see the argument for making it more of a
recommended "next" implementation. Maybe omit this section from the README
until we've got a better concept of our desired workflow?

Reply:
> I had conceptualized our fork of matrix-doc as being more of a staging area
for upstreaming

And this is exactly the purpose, except I only listed point 1 so we can start somewhere.
To me, the overall flow would be something along the lines of:
- document what we find
- think about ourselves if it makes sense. If not talk with Matrix team to figure out if it does or not
- If it does, propose PR
- once Matrix team updated upstream with the authoritative info, we fetch and merge upstream.

I wouldn't think further until we actually got something done.

Reply:
:+1: